bin/queue.bat

```diff
@@ -0,0 +1,11 @@
+@echo off
+
+@setlocal
+
+set YII_PATH=%~dp0
+
+if "%PHP_COMMAND%" == "" set PHP_COMMAND=php.exe
+
+"%PHP_COMMAND%" "%YII_PATH%queue" %*
```

Comment:
Can we remove `%YII_PATH%` and use `%~dp0` directly?

Reply:
done.